src/components/pages/Users/Users.js

```diff
@@ -107,18 +107,27 @@ const NameCellStyle = styled('div')`
 const UserTableTd = styled(Td)`
   position: relative;
 `
-const TableRadioLabel = styled('label')`
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-  position: absolute;
-  display: grid;
-  place-items: center;
-  ${hoverState(css`
-    border: solid 1px ${theme.color.primaryColor};
-  `)}
-`
+const TableRadioLabel = styled.label(
+  (props) => css`
+    top: 0;
+    right: 0;
+    cursor: ${props.cursor};
+    bottom: 0;
+    left: 0;
+    position: absolute;
+    display: grid;
+    place-items: center;
+    ${hoverState(css`
+      border: solid 1px ${theme.color.primaryColor};
+    `)}
+  `,
+)
+
+const TableRadioInput = styled.input(
```

Comment:
both `label` and `input` need a cursor selection - setting it on `label `does not overwrite it on `input`

Reply:
could `input` stay inside `TableRadioLabel` now. It looks much cleaner without passing `getCursorType()` twice into 2 styled components? What do you think? This is up to you.

Reply:
added! good idea